### DISCUSSION TITLE
How to improve snowmelt simulation?

### DISCUSSION THREAD

**Discussion by aiinibxl** [iiasa/CWatM#72] [Q&A]
Hello everyone,

When I use this model to simulate runoff in China, it seems that this model cannot describe the snow and melting process well. For example, in a catchment in northern China, according to the rules in the snow_frost modules, most snow will accumulate until it melts in the spring , but actually snow melts soon after it falls in this region. I tried to modify the temperature threshold, but there was only little improvement.
![sm_xiahui1981-1990](https://github.com/iiasa/CWatM/assets/96671336/c8ced399-0ac4-47c4-aab5-f0fa02191b2e)

Do you have any suggestions for this?

**Comment by aranhax** [comment]
Sorry if this is too late for you, but are you using the option `snowmelt_radiation =  True` ? If yes then you can adjust some parameters such as the `SnowMeltCoef`. This issue might be related to your problem too #110 .
